getMongoData/getMongoData.js

```diff
@@ -302,6 +302,27 @@ function printDataInfo(isMongoS) {
                     }
                     printInfo('Indexes',
                               function(){return db.getSiblingDB(mydb.name).getCollection(col).getIndexes()}, section);
+                    printInfo('Index Stats',
+                              function(){var res = db.getSiblingDB(mydb.name).runCommand( {
```

Comment:
Please fix the indenting here - start a new line for the function contents, and ensure that all lines in this function are aligned.